products/llbuildSwift/BuildDBBindings.swift

```diff
@@ -8,13 +8,13 @@
 
 // This file contains Swift bindings for the llbuild C API.
 
-#if os(macOS)
```

Comment:
Can you please instead write this as `#if canImport(Darwin)`? The `else` case should remain `Glibc` because that's what applies to other platforms as well, like FreeBSD or Android, and this patch would break builds there.

Same for the other places.

Reply:
Done, even added a check for `#canImport(Glibc)` and a custom error when neither of those is available.